tests/explainers/test_linear.py

```diff
@@ -185,6 +186,17 @@ def test_sparse():
     assert np.max(np.abs(scipy.special.expit(explainer.expected_value + shap_values.sum(1)) - model.predict_proba(X)[:, 1])) < 1e-6
 
 
+def test_invalid_feature_perturbation_raises():
+    Ridge = pytest.importorskip("sklearn.linear_model").Ridge
```

Comment:
`sklearn` is one of the package's core dependencies, so `Ridge` can just be imported in the usual way at the top of the module

Reply:
done